crates/ocg_client/src/lib.rs

```diff
@@ -40,10 +46,52 @@ pub fn client_main() {
     // Unset the manifest dir to make bevy load assets from the workspace root
     std::env::set_var("CARGO_MANIFEST_DIR", "");
 
+    let game_config = GameConfig {
+        server: ServerConfig {
+            server_title: String::from("Integrated server"),
+            ..Default::default()
+        },
+    };
+    let game_config = GameConfig::new_handle(game_config);
+    let integ_server = GameServer::new(game_config).expect("Could not start integrated server");
+    integ_server.set_paused(false);
+    let server_pipe = integ_server
+        .create_local_connection()
+        .expect("Could not create an integrated server connection");
+
+    let net_thread = NetworkThread::new(GameSide::Client, NetworkThreadClientState::default);
+    net_thread
+        .exec_async_await(|state| {
+            Box::pin(async move {
+                NetworkThreadClientState::connect_locally(
+                    state,
+                    server_pipe.await.context("integ_server.create_local_connection")?,
+                )
+                .await
+                .context("NetworkThreadClientState::connect_locally")
+            })
+        })
+        .expect("Could not connect the the client to the integrated server")
+        .expect("Could not connect the the client to the integrated server");
```

Comment:
Is it meant to call `expect` a second time? 
Also minor spelling mistake `the the`

Reply:
It is meant to, because of the asynchronicity of this call, the outer Result can have an error from inter-thread communication, and the inner Result has an error from the connection itself.